CreateOwnAdventure

```diff
@@ -0,0 +1,8 @@
+Hello avid travelers. You must be starved. 
+
+And tired.
+
+To explore your dining options, click [here](http://travel.cnn.com/cambodia-best-dishes-cambodia-food-401118/).
+
+To explore your lodging options hosted by Airbnb, click [here](https://www.airbnb.com/help/getting-started/how-it-works).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.